test/E2ETests/Smoke.E2E.Tests/app.config

```diff
@@ -6,6 +6,10 @@
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.2.3.0" newVersion="1.2.3.0" />
       </dependentAssembly>
+      <dependentAssembly>
```

Comment:
Could we automatically generate these using `<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>` in the project?